File: Towers-Of-Hanoi-Part-2/mysolution.js
const hanoi_steps = (number_discs) => {
  step(number_discs, 1, 2, 3);
}

const step = (number, base, empty, goal) => {
  if (number > 0) {
    step(number - 1, base, goal, empty);
    console.log(`${base}->${goal}`);
    step(number - 1, empty, base, goal);
  }
}

hanoi_steps(6);

File: Towers-Of-Hanoi-Part-2/mysolution.py
def hanoi_steps(number_discs):
  step(number_discs, 1, 2, 3)

def step(number, base, empty, goal):
  if(number > 0):
    step(number - 1, base, goal, empty)
    print(f'{base}->{goal}')
    step(number - 1, empty, base, goal)

hanoi_steps(6)